ledger/applications.go

```diff
@@ -110,33 +94,16 @@ func newAppLedger(balances transactions.Balances, acctWhitelist []basics.Address
 	al.appIdx = appIdx
 	al.balances = balances
 	al.schemas = schemas
-	al.addresses = make(map[basics.Address]bool, len(acctWhitelist))
-	al.apps = make(map[basics.AppIndex]bool, len(appGlobalWhitelist))
 	al.AppTealGlobals = globals
-
-	for _, addr := range acctWhitelist {
-		al.addresses[addr] = true
-	}
-
-	for _, aidx := range appGlobalWhitelist {
-		al.apps[aidx] = true
-	}
-
 	return al, nil
 }
 
 // MakeDebugAppLedger returns logic.LedgerForLogic suitable for debug or dryrun
-func MakeDebugAppLedger(balances transactions.Balances, acctWhitelist []basics.Address, appGlobalWhitelist []basics.AppIndex, appIdx basics.AppIndex, schemas basics.StateSchemas, globals AppTealGlobals) (al logic.LedgerForLogic, err error) {
-	return newAppLedger(balances, acctWhitelist, appGlobalWhitelist, appIdx, schemas, globals)
+func MakeDebugAppLedger(balances transactions.Balances, appIdx basics.AppIndex, schemas basics.StateSchemas, globals AppTealGlobals) (al logic.LedgerForLogic, err error) {
+	return newAppLedger(balances, appIdx, schemas, globals)
 }
 
 func (al *appLedger) Balance(addr basics.Address) (res basics.MicroAlgos, err error) {
```

Comment:
Would be very marginally more clear to get rid of these named returned values here.

Reply:
done!